srcpkgs/musl/patches/0001-remove-O_LARGEFILE.patch

```diff
@@ -0,0 +1,26 @@
+From 7d554d0c786f5431ad3035a328815890a95a2979 Mon Sep 17 00:00:00 2001
```

Comment:
Btw, the O_LARGEFILE change specifically affects x86_64, so changing `arch/generic/bits/fcntl.h` wouldn't be appropriate: https://github.com/bminor/musl/commit/b8b729bd22c28c9116c2fce65dce207a35299c26

The appropriate way of handling it would be to add back `arch/x86_64/bits/fcntl.h`: https://github.com/oreo639/void-packages/blob/90536bd5d7dd48c6db6d581ce3054c7dc7b202b6/srcpkgs/musl/patches/x86_64-O_LARGEFILE.patch

Just for future reference, I am looking into musl 1.2.x rn.